rpc/client/http/http.go

```diff
@@ -207,6 +207,7 @@ func (c *baseRPCClient) Status() (*ctypes.ResultStatus, error) {
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
What's with all of these spurious whitespace changes?

Reply:
I just find it nicer with a space above the final return so I add them here and there. I can undo if needed

Reply:
Nah, don't really mind them, but when they're unrelated to the other changes they just make the diff noisier and make it harder to do backports.